KA-Flocking/Assets/Scripts/ToggleSwitch.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ToggleSwitch : MonoBehaviour
+{
+    public Toggle CircularToggle;
+    public Toggle RectangularToggle;
+    public Toggle ArrowToggle;
+    public GameObject CircularToggleOn;
+    public GameObject CircularToggleOff;
+    public GameObject RectangularToggleOn;
+    public GameObject RectangularToggleOff;
+    public GameObject ArrowToggleOn;
+    public GameObject ArrowToggleOff;
+
+    public Toggle InfantryToggle;
+    public Toggle PikeToggle;
+    public Toggle ArcherToggle;
+    public Toggle ScoutToggle;
+    public GameObject InfantryToggleOn;
+    public GameObject InfantryToggleOff;
+    public GameObject PikeToggleOn;
+    public GameObject PikeToggleOff;
+    public GameObject ArcherToggleOn;
+    public GameObject ArcherToggleOff;
+    public GameObject ScoutToggleOn;
+    public GameObject ScoutToggleOff;
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        //Toggle buttons for formation type
+        if (CircularToggle.isOn)
+        {
+            CircularToggleOn.SetActive(true);
+            CircularToggleOff.SetActive(false);
+        }
+        else if (!CircularToggle.isOn)
+        {
+            CircularToggleOn.SetActive(false);
+            CircularToggleOff.SetActive(true);
+        }
```

Comment:
Couldn't this be 
`CirculatToggleOn.SetActive(CircularToggle.isOn);`
`CirculatToggleOff.SetActive(!CircularToggle.isOn);`

Reply:
Ah smart, I'll check it out!